DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Dashboard/Index.cshtml

```diff
@@ -17,214 +18,222 @@
+          Contact number
+        </dt>
+        <dd class="nhsuk-summary-list__value">
+          @Model.CentreDetails.Telephone
+        </dd>
+      </div>
+
+      <div class="nhsuk-summary-list__row details-list-with-button__row">
+        <dt class="nhsuk-summary-list__key">
+          Centre support details
+        </dt>
+        <dd class="nhsuk-summary-list__value">
+          @Model.CentreDetails.BannerText
+        </dd>
+      </div>
+
+      <div class="nhsuk-summary-list__row details-list-with-button__row">
+        <dt class="nhsuk-summary-list__key">
+          Your IP address
+        </dt>
+        <dd class="nhsuk-summary-list__value">
+          @Model.CentreDetails.IpAddress
+        </dd>
+      </div>
+
+      <div class="nhsuk-summary-list__row details-list-with-button__row">
+        <dt class="nhsuk-summary-list__key">
+          Pre-approved IPs
+        </dt>
+        <dd class="nhsuk-summary-list__value">
+          @Model.CentreDetails.ApprovedIps
+        </dd>
+      </div>
+
+    </dl>
+
+    <a class="nhsuk-button" role="button">
+      Edit centre details
+    </a>
   </div>
+</details>
+
+<ul class="nhsuk-grid-row nhsuk-card-group">
+  <li class="nhsuk-grid-column-one-half nhsuk-card-group__item">
+
+    <div class="nhsuk-card nhsuk-card--clickable">
+      <div class="nhsuk-card__content">
+        <h2 class="nhsuk-card__heading nhsuk-heading-m">
+          <a class="nhsuk-card__link" href="#">Contract details</a>
+        </h2>
+        <p class="nhsuk-card__description">See details about your contract type</p>
+      </div>
+    </div>
+
+  </li>
+  <li class="nhsuk-grid-column-one-half nhsuk-card-group__item">
+
+    <div class="nhsuk-card nhsuk-card--clickable">
+      <div class="nhsuk-card__content">
+        <h2 class="nhsuk-card__heading nhsuk-heading-m">
+          <a class="nhsuk-card__link" href="#">Reports</a>
+        </h2>
+        <p class="nhsuk-card__description">See reports about your usage and evaluations</p>
+      </div>
+    </div>
+
+  </li>
+  <li class="nhsuk-grid-column-one-half nhsuk-card-group__item">
+
+    <div class="nhsuk-card nhsuk-card--clickable">
+      <div class="nhsuk-card__content">
+        <h2 class="nhsuk-card__heading nhsuk-heading-m">
+          <a class="nhsuk-card__link" href="#">Top courses</a>
+        </h2>
+        <p class="nhsuk-card__description">See information about your top courses</p>
+      </div>
+    </div>
+
+  </li>
+  <li class="nhsuk-grid-column-one-half nhsuk-card-group__item">
+
+    <div class="nhsuk-card nhsuk-card--clickable">
+      <div class="nhsuk-card__content">
+        <h2 class="nhsuk-card__heading nhsuk-heading-m">
+          <a class="nhsuk-card__link" href="#">Centre ranking</a>
+        </h2>
+        <p class="nhsuk-card__description">See centre rankings by week, month, or year</p>
+      </div>
+    </div>
+
+  </li>
+</ul>
+
+<nav class="side-nav-menu-bottom" aria-label="Bottom navigation bar">

```

Comment:
Lots of line ending and indentation changes on this file. Only new addition is this new nav and another highlighted change near the top of the page.